patches/core-setup/0008-add-intermediate-packages.patch

```diff
@@ -0,0 +1,106 @@
+From fdf6d172cb103288b16f8bfcd5fc1df69783e907 Mon Sep 17 00:00:00 2001
+From: karajas <karajas@microsoft.com>
+Date: Tue, 5 Sep 2017 17:36:32 -0400
+Subject: [PATCH] Add core-setup patch to allow intermediate package cache to pick up source built packages 
+
+---
+ dir.props                                        |  3 ++-
+ src/sharedFramework/framework/framework.csproj   |  4 ++++
+ src/sharedFramework/lockedhost/lockedhost.csproj |  3 +++
+ src/sharedFramework/sharedFramework.proj         | 16 ++++++++--------
+ 4 files changed, 17 insertions(+), 9 deletions(-)
+
+diff --git a/dir.props b/dir.props
+index 2909110..5ae64a9 100644
+--- a/dir.props
++++ b/dir.props
+@@ -276,6 +276,7 @@
+ 
+     <SharedFrameworkPublishDir>$(IntermediateOutputRootPath)sharedFrameworkPublish\</SharedFrameworkPublishDir>
+     <SharedFrameworkPublishSymbolsDir>$(IntermediateOutputRootPath)sharedFrameworkPublish.symbols\</SharedFrameworkPublishSymbolsDir>
++    <SharedFrameworkIntermediatePackagesDir>$(IntermediateOutputRootPath)sharedFrameworkPublish\packages\</SharedFrameworkIntermediatePackagesDir>
+     <SharedFrameworkNameAndVersionRoot>$(SharedFrameworkPublishDir)shared\$(SharedFrameworkName)\$(SharedFrameworkNugetVersion)</SharedFrameworkNameAndVersionRoot>
+   </PropertyGroup>
+ 
+@@ -429,4 +430,4 @@
+   <!-- Use Roslyn Compilers to build -->
+   <Import Project="$(RoslynPropsFile)" Condition="'$(OsEnvironment)'!='Windows_NT' and Exists('$(RoslynPropsFile)') and '$(UseRoslynCompilers)'!='false'" />
+   <Import Project="$(RoslynPropsFile)" Condition="'$(OsEnvironment)'=='Windows_NT' and Exists('$(RoslynPropsFile)')" />
+-</Project>
+\ No newline at end of file
++</Project>
+diff --git a/src/sharedFramework/framework/framework.csproj b/src/sharedFramework/framework/framework.csproj
+index 52e6553..d4bcf1d 100644
+--- a/src/sharedFramework/framework/framework.csproj
++++ b/src/sharedFramework/framework/framework.csproj
+@@ -4,4 +4,8 @@
+     <OutputType>Exe</OutputType>
+   </PropertyGroup>
+ 
++  <ItemGroup>
++    <PackageReference Include="Microsoft.NETCore.Platforms" Version="2.0.0" />
```

Comment:
Why is this necessary?  It shouldn't be.  This project already reference Microsoft.NETCore.App, which references this package.

Reply:
This isn't necessary but adding a direct reference, helped me understand which Platforms was being included in the assets file in an isolated restore. I can remove this.

Reply:
Yes, please remove this.